chain/beacon/node.go

```diff
@@ -51,11 +53,12 @@ type Handler struct {
 	serving bool
 	stopped bool
 	l       log.Logger
+	version utils.Version
 }
 
 // NewHandler returns a fresh handler ready to serve and create randomness
 // beacon
-func NewHandler(c net.ProtocolClient, s chain.Store, conf *Config, l log.Logger) (*Handler, error) {
+func NewHandler(c net.ProtocolClient, s chain.Store, conf *Config, l log.Logger, version utils.Version) (*Handler, error) {
```

Comment:
Why is this in "utils" and not in "common" (both are new packages right ?) ? It feels better to put this in common since well the version is common to the whole codebase and it's not an utility function ?